src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php

```diff
@@ -567,4 +553,24 @@ private function handleAdaptThemeToRTLLanguagesException(ThemeException $e)
 
         return $this->getFallbackErrorMessage($type, $e->getCode());
     }
+
+    /**
+     * Gets exception or exception and its code error mapping.
+     *
+     * @param ShopException $exception
+     *
+     * @return array
+     */
+    private function getLogoUploadErrorMessages(ShopException $exception)
+    {
+        return [
+            NotSupportedLogoImageExtensionException::class =>
+                $this->trans('Image format not recognized, allowed formats are: .gif, .jpg, .png', 'Admin.Notifications.Error'),
```

Comment:
Hi @LouiseBonnard - the message "Image format not recognized, allowed formats are: .gif, .jpg, .png"  seems to be incorrect for me due to it forces several image formats only. As I checked in my case this message is even incorrect due to there are more supported types.

Im thinking about just adding the translation "Image format not recognized, allowed formats are: %allowed_formats%"
or "Image format not recognized, allowed formats are: %s"

Reply:
Indeed! We could write _Image format not recognized, allowed format(s) is(are): .%s_ in _Admin.Notifications.Error_ like suggested in two different pull requests, cf. https://github.com/PrestaShop/PrestaShop/pull/13485#discussion_r277726738. What do you think?

Reply:
agree. Done :)